clash-vexriscv/src/ffi/impl.cpp

```diff
@@ -146,12 +163,15 @@ void vexr_step_rising_edge(VVexRiscv *top, uint64_t time_add, const NON_COMB_INP
 
 	top->clk = true;
 	top->eval();
+	if (vcd != NULL){
+		vcd->dump (contextp->time());
```

Comment:
That's some Haskell formatting right here :sweat_smile:  Same in the falling edge function
```suggestion
	if (vcd != NULL) {
		vcd->dump(contextp->time());
```